src/helm/proxy/clients/auto_client.py

```diff
@@ -66,9 +68,17 @@ def _get_client(self, model_deployment_name: str) -> Client:
             host_organization: str = model_deployment.host_organization
             cache_config: CacheConfig = build_cache_config(self.cache_path, self.mongo_uri, host_organization)
 
+            # Initialize `FileCache` for text-to-image model APIs
+            local_file_cache_path: str = os.path.join(self.cache_path, "output", host_organization)
+            file_cache: FileCache = LocalFileCache(local_file_cache_path, file_extension="png")
```

Comment:
Make `file_cache` a `provider_binding` rather than a `constant_binding` i.e. lazily create the folder in a lambda.

Reply:
Do you mind explaining `provider_binding` vs `constant_binding` a bit more and sharing how the lambda looks like?

Reply:
For `provider_binding`, the value should be a parameter-less function that returns the value, rather than the value itself. It can either be a lambda or a named method.

Reply:
I'll send you a PR for this.

Reply:
Opened PR #2175 for this.